inc/mlx90632.h

```diff
@@ -157,10 +171,14 @@
 #define MLX90632_REF_3  12.0 /**< ResCtrlRef value of Channel 3 */
 #define MLX90632_XTD_RNG_KEY 0x0500 /**Extended range support indication key */
 
-/* Measurement types */
-#define MLX90632_MTYP_MEDICAL 0
-#define MLX90632_MTYP_EXTENDED 17
+/* Measurement types - the MSBit is for software purposes only and has no hardware bit related to it. It indicates continious '0' or sleeping step burst - '1' measurement mode*/
+#define MLX90632_MTYP_MEDICAL 0x00
+#define MLX90632_MTYP_EXTENDED 0x11
+#define MLX90632_MTYP_MEDICAL_BURST 0x80
+#define MLX90632_MTYP_EXTENDED_BURST 0x91
+#define MLX90632_BURST_MEASUREMENT_TYPE(meas_type) meas_type + 0x80 /**<The MSBit is only used in the software to indicate burst type measurement. The 5 LS Bits define medical ot extended measurement and are used to set the hardware*/
```

Comment:
```suggestion
#define MLX90632_BURST_MEASUREMENT_TYPE(meas_type) meas_type + 0x80 /**<The MSBit is only used in the software to indicate burst type measurement. The 5 LS Bits define medical or extended measurement and are used to set the hardware */
#define MLX90632_BURST_MEASUREMENT_TYPE_STATUS(mtyp_type) mtyp_type & 0x7F /**< Extract the measurement type from MTYP */
```

Reply:
Maybe you have better name than above.

Reply:
I think it should be without the BURST part as this actually removes the cont/burst indication bit and leaves only the medical/extended part. So maybe something like MLX90632_MEASUREMENT_TYPE_STATUS?

Reply:
Updated with your suggestion.

Reply:
```suggestion
#define MLX90632_BURST_MEASUREMENT_TYPE(meas_type) meas_type + 0x80 /**<The MSBit is only used in the software to indicate burst type measurement. The 5 LS Bits define medical or extended measurement and are used to set the hardware */

```

Reply:
The typo here is still there. I will commit my suggestion.